dist/tools/suit_v4/gen_manifest.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2019 Inria
+#               2019 FU Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+import os
+import hashlib
+import json
+import uuid
+
+import click
```

Comment:
I'm fine if you like the click framework, but given the (lack of) complexity of this script, is it really justified to add one more dependency?

Reply:
I addressed this comment in 0440c646913fb37598d70572429ce7aa73ef80b0, hope this is fine.